specifyweb/frontend/js_src/lib/components/Header/index.tsx

```diff
@@ -221,7 +221,7 @@ export function MenuButton({
     p-[1.4vh]
     ${
       isActive
-        ? 'bg-brand-300 text-white'
+        ? 'bg-brand-300 !text-white'
```

Comment:
(optionally) find out what causes this to become an issue now because it wasn't an issue before

Reply:
I couldn't find the reason :confused: